config/sync/taxonomy.vocabulary.opportunity_type.yml

```diff
@@ -18,7 +18,7 @@ third_party_settings:
     unpublish_enable: false
     unpublish_required: false
     unpublish_revision: false
-name: 'Opportunity Type'
+name: 'Stanford Opportunity Type'
```

Comment:
Can you change the CS view instead to prevent any inheritance issues?

Reply:
@pookmish I named it back.